### PR TITLE
Configuring initial structural markup.

### LINKED ISSUE #2364: [SMS Campaign] Add initial structural markup

### PR DESCRIPTION
**Pull request by weerd**
Resolves #2364 

@barryclark @aaronschachter 

Adjusting the initial structural markup to comply with new `container` module class stuffs. Will be edited further, but this is the initial outer structure stuff.

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--sms-game.tpl.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
  */
 ?>
 
-<section class="campaign--wrapper action">
+<article id="" class="">
```
Comment (barryclark) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--sms-game.tpl.php:13]: @weerd could you explain why you've included empty id and class attributes?

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--sms-game.tpl.php:13]: Yep! Cause I'll be adding ids and classes in the next stage, but need to determine what those are. I was mostly concentrating on the outer structure for this PR ;)

**Comment by barryclark** [comment]
Looks good! :sparkles: :sparkles: :sparkles:

**Comment by weerd** [comment]
:boom:

**Pull request merged**